### ISSUE TITLE
geth fails to build on OpenBSD 7.2 amd64

### ISSUE THREAD

**Issue by essays-on-esotericism** [ethereum/go-ethereum#26800]
#### System information

Geth version: v1.11.2
OS & Version: OpenBSD 7.2 amd64

#### Expected behaviour
geth successfully compiles using 'make install' or 'go install ...'

#### Actual behaviour
geth fails to build

#### Steps to reproduce the behaviour
Attempt to build geth on OpenBSD:

$ go install github.com/ethereum/go-ethereum/cmd/geth@latest

OR

$ git clone https://github.com/ethereum/go-ethereum
$ cd go-ethereum/
$ git checkout v1.11.2
$ make install

#### Backtrace

````
$ go install github.com/ethereum/go-ethereum/cmd/geth@latest
go: downloading github.com/ethereum/go-ethereum v1.11.2
go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.2
go: downloading github.com/russross/blackfriday/v2 v2.1.0          
# github.com/karalabe/usb    
In file included from ../../go/pkg/mod/github.com/karalabe/usb@v0.0.2/raw_errors.go:22:
../../go/pkg/mod/github.com/karalabe/usb@v0.0.2/./libusb/libusb/libusb.h:1807:67: warning: declaration of 'struct timeval' will not be visible outside of this function [-Wvisibility]
../../go/pkg/mod/github.com/karalabe/usb@v0.0.2/./libusb/libusb/libusb.h:1810:9: warning: declaration of 'struct timeval' will not be visible outside of this function [-Wvisibility]
../../go/pkg/mod/github.com/karalabe/usb@v0.0.2/./libusb/libusb/libusb.h:1812:9: warning: declaration of 'struct timeval' will not be visible outside of this function [-Wvisibility]
../../go/pkg/mod/github.com/karalabe/usb@v0.0.2/./libusb/libusb/libusb.h:1816:9: warning: declaration of 'struct timeval' will not be visible outside of this function [-Wvisibility]
../../go/pkg/mod/github.com/karalabe/usb@v0.0.2/./libusb/libusb/libusb.h:1819:9: warning: declaration of 'struct timeval' will not be visible outside of this function [-Wvisibility]
# github.com/cockroachdb/pebble/vfs
../../go/pkg/mod/github.com/cockroachdb/pebble@v0.0.0-20230209160836-829675f94811/vfs/disk_usage_unix.go:18:27: stat.Bsize undefined (type unix.Statfs_t has no field or method Bsize)
../../go/pkg/mod/github.com/cockroachdb/pebble@v0.0.0-20230209160836-829675f94811/vfs/disk_usage_unix.go:18:48: stat.Bfree undefined (type unix.Statfs_t has no field or method Bfree)
../../go/pkg/mod/github.com/cockroachdb/pebble@v0.0.0-20230209160836-829675f94811/vfs/disk_usage_unix.go:19:28: stat.Bsize undefined (type unix.Statfs_t has no field or method Bsize)
../../go/pkg/mod/github.com/cockroachdb/pebble@v0.0.0-20230209160836-829675f94811/vfs/disk_usage_unix.go:19:49: stat.Bavail undefined (type unix.Statfs_t has no field or method Bavail)
../../go/pkg/mod/github.com/cockroachdb/pebble@v0.0.0-20230209160836-829675f94811/vfs/disk_usage_unix.go:20:28: stat.Bsize undefined (type unix.Statfs_t has no field or method Bsize)
../../go/pkg/mod/github.com/cockroachdb/pebble@v0.0.0-20230209160836-829675f94811/vfs/disk_usage_unix.go:20:49: stat.Blocks undefined (type unix.Statfs_t has no field or method Blocks)

````

When submitting logs: please submit them as text and not screenshots.

**Comment by karalabe** [comment]
Hmm, known issue upstream at Pebble: https://github.com/cockroachdb/pebble/issues/2341

I guess we should disable Pebble on OpenBSD until upstream support lands.

**Comment by karalabe** [comment]
This should be fixed on master, will be released in Geth 1.11.3 maybe next week of the one after.
